.ci/appveyor.yml

```diff
@@ -63,7 +71,8 @@ test_script:
     "%CMD_IN_ENV% python -m pytest
     --cov -k "not ClangASTPrintBear and not ClangCloneDetectionBear and
     not ClangComplexityBear and not ClangCountVectorCreator and
-    not ClangCountingConditions"
+    not ClangCountingConditions and not RuboCopBearTest and
```

Comment:
It looks like the Rubocop tests needs rework for Windows? It's harder to read without colors :/